packages/http-client-csharp/generator/.editorconfig

```diff
@@ -0,0 +1,204 @@
+# editorconfig.org
+
+# top-most EditorConfig file
+root = true
```

Comment:
this technically isn';t the root config, could at least remove the options we already declare in the top level one no?

Reply:
This keeps it from finding anything at the higher level so its needed for isolation